doc/devel/documenting_mpl.rst

```diff
@@ -8,8 +8,14 @@ Getting started
 ===============
 
 The documentation for matplotlib is generated from ReStructured Text using the
-Sphinx_ documentation generation tool. Sphinx-1.0 or later and numpydoc 0.4 or
-later is required.
+Sphinx_ documentation generation tool.
+The requirements are as follows (https://github.com/matplotlib/matplotlib/blob/master/doc-requirements.txt)
```

Comment:
I changed it. Now it looks like a proper list. Sorry for any inconvenience caused